layout/modals/popups/speedometer-select.xml

```diff
@@ -22,8 +22,9 @@
 		<Panel id="SpeedometerSelectContainer" class="flow-down horizontal-align-center">
 		</Panel>
 		<Panel class="row horizontal-align-center mt-4">
-			<TextEntry id="SpeedometerName" class="textentry" maxchars="255" placeholder="#Settings_Speedometer_Customname_Placeholder" text="" />
+			<TextEntry id="SpeedometerName" class="textentry" maxchars="255" placeholder="#Settings_Speedometer_Customname_Placeholder" text="" ontextentrysubmit="SpeedometerSelectPopup.onTextSubmitted()"/>
 		</Panel>
+		<Label id="InvalidNameLabel" class="text-m horizontal-align-center" text="#Settings_Speedometer_Customename_InvalidName" />
```

Comment:
Actually, this name isn't great. Let's go with `Settings_Speedometer_InvalidName`.

Reply:
all set!